backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -408,6 +408,27 @@ func (ctx *AppContext) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// MyMasterPubKeys returns a map of master pubkeys to their account
```

Comment:
I'd prefer to change the possessive language.  i.e. "My" and "their" could be changed to more descriptive things.  From the following code, it seems the function retrieves the xpub keys for each account, excluding imported.

In api.proto, `MyMasterPubKeys` would also be better named just `MasterPubKeys` or `WalletMasterPubKeys`.